service/topology_coordinator.cc

```diff
@@ -1577,7 +1577,30 @@ class topology_coordinator : public endpoint_lifecycle_subscriber {
         rtlogger.info("entered `{}` transition state", *tstate);
         switch (*tstate) {
```

Comment:
> Queries that are sent before that will fail
> because schema is empty, so they will not find any tables in the first
> place. The is pre-existing and not addressed by this patch.

Is this what I described here https://github.com/scylladb/scylladb/issues/19093#issuecomment-2149454983?

I'd like to understand why queries are failing due to empty schema, instead of blocking on schema sync read barrier.

Reply:
I did not look dipper, but `get_schema_for_write` gets table_schema_version, not global schema version. Probably we do not find the table that corresponds to the schema.